src/stirling/bpf_tools/bcc_wrapper.h

```diff
@@ -283,13 +298,11 @@ class BCCWrapperImpl : public BCCWrapper {
     PX_RETURN_IF_ERROR(bpf_.open_perf_event(table_name, type, config));
     return Status::OK();
   }
-  void PollPerfBuffers(int timeout_ms = 0) override;
-  void PollPerfBuffer(std::string_view perf_buffer_name, int timeout_ms) override;
+  void PollPerfBuffers(const int timeout_ms = 0) override;
+  Status PollPerfBuffer(const std::string& name, const int timeout_ms = 0) override;
   void Close() override;
 
- protected:
   Status ClosePerfBuffer(const PerfBufferSpec& perf_buffer) override;
-  std::vector<PerfBufferSpec> perf_buffers_;
```

Comment:
It seems like these changes aren't specifically related to the record and replay code:
* perf_buffers_ -> perf_buffers_spec rename
* Function signature changes to use `const` and `std::string_view`

Could we separate these out into another PR since the `bcc_wrapper.*` changes are some of the largest? Maybe we can introduce the `CommonPerfBufferSetup` construct without using it as well to further reduce the lines of code?

Reply:
Done in #1697. Thank you!